style/preview.less

```diff
@@ -23,17 +24,23 @@
 		}
 
 		&-wordmark {
-			background-image: inline-svg( '../images/wikipedia-wordmark.svg' );
+			background-image: inline-svg( '../images/wikipedia-wordmark-en.svg' );
 			background-repeat: no-repeat;
-			background-color: #eaf3ff;
 			background-position: left;
 			background-position-x: 11px;
 			height: 100%;
 			flex-grow: 1;
 			font-size: 24px;
-			padding-left: 13px;
+			margin-left: 13px;
 			color: #72777d;
 			font-family: 'Linux Libertine', 'Georgia', 'Times', serif;
```

Comment:
no longer need to font related style - size, color, family, so safe to delete

Reply:
right